circle.yml

```diff
@@ -420,6 +410,12 @@ jobs:
       - build
       - download_execution_tests:
           rev: v17.0
+      - run:
+          name: "Filter-out invalid Prague tests"
+          working_directory: ~/tests
+          command: |
+            sudo apt-get -q update && sudo apt-get -qy install jq
+            find BlockchainTests -name '*.json' -exec sh -c 'jq "with_entries(select(.key | test(\"Prague\") | not))" {} > {}.tmp && mv {}.tmp {}' \;
```

Comment:
what does this do? filters out all Prague tests?

Reply:
Yes, removes all tests with "Prague" in the name. They are not updated and fail because they are missing system contracts.